File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.cc
#include <iostream>
using namespace std;

int main() {
	
	int s, k, h;
	cin >> s >> k >> h;

	if (s + k + h >= 100) 
	{
		cout << "OK" << endl;
	}
	else if (min(s, min(k, h)) == s)
	{
		cout << "Soongsil" << endl;
	}
	else if (min(s, min(k, h)) == k)
	{
		cout << "Korea" << endl;
	}
	else {
		cout << "Hanyang" << endl;
	}
}

File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int s = sc.nextInt();
        int k = sc.nextInt();
        int h = sc.nextInt();

        if (s + k + h >= 100) {
            System.out.println("OK");
        } else {
            if (Math.min(s, Math.min(k, h)) == s) {
                System.out.println("Soongsil");
            } else if (Math.min(s, Math.min(k, h)) == k) {
                System.out.println("Korea");
            } else {
                System.out.println("Hanyang");
            }
        }
    }
}